MapboxNavigation/PollyVoiceController.swift

```diff
@@ -117,7 +116,11 @@ public class PollyVoiceController: RouteVoiceController {
             input.voiceId = voiceId
         }
         
-        input.text = "<speak><amazon:effect name=\"drc\"><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>\(text)</prosody></amazon:effect></speak>"
+        let wrappedSSML = text
+            .replacingOccurrences(of: "^<speak\\s*(?:\\s+[^>]*)?>", with: "$0<amazon:effect name=\"drc\"><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>", options: .regularExpression)
+            .replacingOccurrences(of: "</speak>$", with: "</prosody></amazon:effect></speak>",options: .regularExpression)
```

Comment:
Noting for posterity that we need to remove this code once the Directions API begins to add these tags itself.

Reply:
@1ec5 what do you think about making this future proof-able and adding an xml parsing dependency? It might make sense to check and make sure we have all necessary keys.

Reply:
No, I think we should introduce an XML parsing library only when we need it for a particular purpose, so that we can evaluate each candidate library based on how well it’d handle our use case. As it stands, we’re planning to have the Directions API add these tags, so the SDK won’t have any XML parsing needs for now.